samples/nrf9160/cell_pos/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
```

Comment:
```suggestion
# Copyright (c) 2021 Nordic Semiconductor
```
New files should probably have year 2021 in the copyright. Check other files as well.